src/common/communityLevel.ts

```diff
@@ -0,0 +1,26 @@
+export enum CommunityLevel {
```

Comment:
I think it should be named `community_level.ts` to follow our recent convention?

Reply:
oh yes, true!